src/lib/bootloader/udev_mapping.rb

```diff
@@ -23,6 +23,9 @@ class << self
     # @param dev [String] device udev, mdadm or kernel name like /dev/disk/by-id/blabla
     # @raise when device have udev format but do not exists
     def to_kernel_device(dev)
+      # AutoYaST configuration mode. There is no access to the system
+      return nil if Yast::Mode.config
```

Comment:
We should document the `@return` value (I know it wasn't documented before, but it would be nice).